pokemongo_bot/cell_workers/evolve_pokemon.py

```diff
@@ -66,12 +66,15 @@ def work(self):
                 pokemon_to_be_evolved = pokemon_to_be_evolved + min(candy.quantity / (pokemon.evolution_cost - 1), filtered_dict[pokemon.pokemon_id])
 
         if pokemon_to_be_evolved >= self.min_pokemon_to_be_evolved:
+            self.logger.info("Checking minimum pokemon to evolve: {}/{}. Hooray!".format(pokemon_to_be_evolved, self.min_pokemon_to_be_evolved))
```

Comment:
I just think it's valid to state two versions of the same message so people know if something's wrong